sdk/include/ndk/mmtypes.h

```diff
@@ -208,7 +212,6 @@ typedef enum _MI_VAD_TYPE
 } MI_VAD_TYPE, *PMI_VAD_TYPE;
 
 #ifdef NTOS_MODE_USER
-
```

Comment:
Why this change? (removing a newline here, but you added one in a similar case above)

Reply:
Long block: 1 matching line.
Short block: 0 matching line.

Reply:
This doesn't make much sense.